src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (a) {
+      this.result += a;
+    },
+
+    subtract: function (a) {
+      this.result -= a;
+    },
+
+    multiply: function (a) {
+      this.result *= a;
+    },
+
+    divide: function (a) {
+      if (a !== 0) {
+        this.result /= a;
+      } else {
+        return 'Cannot divide by zero';
```

Comment:
The `divide` method returns a string 'Cannot divide by zero' when division by zero is attempted. This is inconsistent with other methods that do not return a value. Consider handling this case differently, such as logging an error or throwing an exception, to maintain consistency in method behavior.